scripts/deploy_nightly.sh

```diff
@@ -10,7 +10,6 @@ set -x				# echo commands
 # By default, github actions overrides the credentials used to access any
```

Comment:
Could you move the comment to the actions file as well?

Reply:
Done.